dev/com.ibm.ws.wsoc/src/com/ibm/ws/wsoc/WsocUpgradeHandler.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2017, 2022 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
```

Comment:
Why the copyright update, did we make changes here previously and forget to update the copyright?